third_party/terraform/utils/convert.go

```diff
@@ -28,6 +28,26 @@ func Convert(item, out interface{}) error {
 	return nil
 }
 
+// When converting to a map, we can't use setOmittedFields because FieldByName
+// fails. Luckily, we don't use the omitted fields anymore with generated
+// resources, and this function is used to bridge from handwritten -> generated.
+// Since this is a known type, we can create it inline instead of needing to
+// pass an object in.
+func ConvertToMap(item interface{}) (map[string]interface{}, error) {
```

Comment:
@nstogner looks like you'll be pulling this in soon and won't have to use https://github.com/GoogleCloudPlatform/magic-modules/pull/1496/files#diff-218e98d5cf92f706a23503eda8bbbdfdR10 any more.